tensorflow_lite_support/ios/task/audio/core/audio_record/sources/TFLAudioRecordErrorCode.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * @enum TFLAudioRecordErrorCode
+ * This enum specifies  error codes for TFLAudioRecord of the TensorFlow Lite Task Library.
+ */
+typedef NS_ENUM(NSUInteger, TFLAudioRecordErrorCode) {
```

Comment:
Please move this to the TFLAudioRecord.h file because it's an integral part of the AudioRecord API.